mesonbuild/msubprojects.py

```diff
@@ -65,23 +96,23 @@ def done(self, wrap_name: str, log_queue: T.List[T.Tuple[mlog.TV_LoggableList, T
 
 
 class Runner:
-    def __init__(self, logger: Logger, r: Resolver, wrap: PackageDefinition, repo_dir: str, options: argparse.Namespace) -> None:
+    def __init__(self, logger: Logger, r: Resolver, wrap: PackageDefinition, repo_dir: str, options: 'Arguments') -> None:
         # FIXME: Do a copy because Resolver.resolve() is stateful method that
         # cannot be called from multiple threads.
         self.wrap_resolver = copy.copy(r)
-        self.wrap_resolver.current_subproject = 'meson'
```

Comment:
Is this right?

Reply:
It was correct until I removed the code which accessed `.current_subproject` a week later in commit f52a5a7cd3f30645c1b746488ec921451545d9b6.

Now it is just unneeded...